docs/user-guide/work-packages/work-package-relations-hierarchies/README.md

```diff
@@ -107,9 +107,9 @@ To display work package hierarchies make sure the *Subject* column is displayed.
 
 You can also add a column with information about parent work packages:
 
-1. In the work package settings menu, click on **Columns**.
-2. Use auto-completion to search and add the *Parent* column.
-3. Click on **Apply** to display the new parent column in the work package list.
+1. Click on the gear wheel in the upper right corner of the work package table to open the work package table configuration.
+2. Select "Parent" from the drop down menu and click on apply.
+3. Now the column with the information about a parant relation is displayed in the work package table.
 
-![parent](image-20200129145338301.png)
+![change-parent-work-package](../../../../../../../change-parent-work-package.png)
```

Comment:
The image path is not right. 
(The filename itself is the same as the image above. Intentional?)